src/unified_recon/rule_0/main.py

```
@@ -0,0 +1,374 @@
+        # Display results
+        display_name = exchange_name.replace("_match", "").upper()
+        display = UnifiedDisplay(display_name, tolerance_dict)
+        display.show_header()
+        display.show_summary(stats)
+        display.show_comparison_by_product(comparisons)
+        
+        # Show detailed breakdown if requested
+        if show_details:
+            display.show_position_details(trader_matrix, exchange_matrix)
+        
+        return stats
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)